src/pages/login.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseComponent } from '../shared/ui/BaseComponent';
+import { isLoggedIn, login } from '../shared/util/auth';
+import { router } from '../shared/util/Router';
+
+const ID = {
+  LOGIN_FORM: 'login-form',
+  USER_NAME: 'username',
+};
+
+export default class LoginPage extends BaseComponent {
+  afterRender() {
+    this.checkAccess();
+    this.bindLoginEvent();
+  }
+
+  private checkAccess() {
+    if (isLoggedIn()) {
+      router.navigateTo('/');
+    }
+  }
+
+  private bindLoginEvent() {
+    const $loginForm = this.$root?.querySelector<HTMLFormElement>(
+      `#${ID.LOGIN_FORM}`
+    );
+    if (!$loginForm) return;
```

Comment:
저는 if문의 조건물을 사용할 때 논리적 NOT 연산자(!)를 최대한 회피하려고 하는데요. 이유는 개발자가 조건을 읽을 때, 2번 생각해야 하기 때문이에요. loginForm의 기본 상태를 한 번 생각해보고, 그 값을 뒤집는 반대 상태를 다시 한 번 생각해봐야 하는데요.
개발자가 코드를 읽을 때, 생각을 한 번이라도 줄여줄 수 있는 로직에 큰 매력이 있다고 생각해요(개발자의 뇌를 가볍게 만들어줍니다).

```suggestion
    if ($loginForm == null) return;
```
이런식으로 엄격한 동등 연산자가 아닌 동등 연산자를 활용하면 쉽게 구현할 수 있습니다. 정원님은 제 의견에 대해 어떻게 생각하고 계신지 궁금합니다!

Reply:
와우! 정말 생각해보지 못한 부분이었는데, 좋은 리뷰 감사드립니다 :)

저도 동규님 의견에 공감해요. 같은 결과를 얻을 수 있는 코드라면, 다른 개발자가 더 직관적으로 이해할 수 있는 방식으로 작성하는 것이 훨씬 더 좋다고 생각해요.

말씀하신 부분 중에서 특히 `!` 연산자가 기본 상태를 연상시킬 수 있다는 점이 저에게도 새로운 관점이었어요. 그 점을 고려하면 동규님 제안이 더 명확해 보이네요.

이 부분은 내일 꼭 코드에 반영하도록 하겠습니다! 다시 한 번 감사드려요 😆